finalitygadget/finalitygadget.go

```diff
@@ -252,46 +252,22 @@ func (fg *FinalityGadget) QueryBlockRangeBabylonFinalized(
 	return finalizedBlockHeight, nil
 }
 
-/* QueryBtcStakingActivatedTimestamp returns the timestamp when the BTC staking is activated
- *
- * - We will check for k deep and covenant quorum to mark a delegation as active
- * - So technically, activated time needs to be max of the following
- *	 - timestamp of Babylon block that BTC delegation receives covenant quorum
- *	 - timestamp of BTC block that BTC delegation's staking tx becomes k-deep
- * - But we don't have a Babylon API to find the earliest Babylon block where BTC delegation gets covenant quorum.
- *   and it's probably not a good idea to add this to Babylon, as this creates more coupling between Babylon and
- *   consumer. So waiting for covenant quorum can be implemented in a clean way only with Babylon side support.
- *   this will be considered as future work
- * - For now, we will use the k-deep BTC block timestamp for the activation timestamp
- * - The time diff issue is not burning. The time diff between pending and active only matters if FP equivocates
- *   during that time period
- *
- * returns math.MaxUint64, ErrBtcStakingNotActivated if the BTC staking is not activated
- */
+// QueryBtcStakingActivatedTimestamp retrieves BTC staking activation timestamp from the database
+// returns math.MaxUint64, error if any error occurs
```

Comment:
Probably this function can be renamed as it no longer queries BBN node?

Reply:
it is an [interface function](https://github.com/babylonlabs-io/finality-gadget/blob/main/finalitygadget/interface.go#L56), renaming is not needed although it doesn't query the BBN node